# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../e2fsprogs/elf-lib-symlinks.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The manaully crafted symlinks must not be absolute ones, especially
not for cross builds, ...

  - Rene Rebe <rene@exactcode.de>

--- e2fsprogs-1.41.0/lib/Makefile.elf-lib.vanilla	2008-07-24 14:53:14.000000000 +0200
+++ e2fsprogs-1.41.0/lib/Makefile.elf-lib	2008-07-24 14:55:58.000000000 +0200
@@ -42,11 +42,10 @@
 install-shlibs install:: $(ELF_LIB) installdirs-elf-lib
 	@echo "	INSTALL-ELF-LIB $(ELF_INSTALL_DIR)/$(ELF_LIB)"
 	@$(INSTALL_PROGRAM) $(ELF_LIB) $(DESTDIR)$(ELF_INSTALL_DIR)/$(ELF_LIB)
-	@echo "	SYMLINK $(ELF_INSTALL_DIR)/$(ELF_SONAME)"
-	@$(LN_S) -f $(ELF_LIB) $(DESTDIR)$(ELF_INSTALL_DIR)/$(ELF_SONAME)
-	@echo "	SYMLINK $(libdir)/$(ELF_IMAGE).so"
-	@$(LN_S) -f $(ELF_INSTALL_DIR)/$(ELF_SONAME) \
-		$(DESTDIR)$(libdir)/$(ELF_IMAGE).so
+	@echo "	SYMLINK $(ELF_SONAME)"
+	@$(LN_S) -f $(ELF_LIB) $(DESTDIR)$(libdir)/$(ELF_SONAME)
+	@echo "	SYMLINK $(ELF_IMAGE).so"
+	@$(LN_S) -f $(ELF_SONAME) $(DESTDIR)$(libdir)/$(ELF_IMAGE).so
 	@echo "	LDCONFIG"
 	@-$(LDCONFIG)
 
